app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,99 @@
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Making maths add up",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>Maths is critical to science, technology and engineering as well as being fundamental for most jobs.</p>
+<p>A good understanding of maths has a lot of benefits for young people to prepare them for their future, like using financial maths and data handling.</p>
+<p>It's also our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.</p>
+    <% end %>
+  </section>
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+
+      <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">national curriculum</a>.</p>
+      <p>When teaching 11 to 16 year olds (key stage 3 and 4), you can use interactive games with conkers and dice to help your pupils understand algebra. Themes you’ll cover are:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Numbers</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+      <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "I try to find out what they’re interested in, whether it’s card games, football or whatever, and use this to make maths relevant.",
+    name: "Dimitra, maths teacher",
+    large: true
+  ) %>
+</section>
+    <p><a href="https://www.thenational.academy/teachers/programmes/maths-secondary-ks3-l/units">Explore what teaching a maths lesson would be like</a>.</p>
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
```

Comment:
@gemmadallmandfe I wasn't sure if Maths Lead should be lower case? We seem to be a bit inconsistent as it's capped in some places on blog posts and not others?

Reply:
I think probably yes - but happy to go with whatever you think (we should review across the site for consistently at some point)

would suggest simplifying a bit further too - `maths lead, London` ?